config/deploy.rb

```diff
@@ -44,3 +44,79 @@
     end
   end
 end
+
+namespace :container do
+  desc 'Hello, world!'
+  task :hello do
+    on roles(:container) do
+      execute "echo Hello, world!"
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)